### PR TITLE
manager integration of Public Cloud Load Balancer

### PR DESCRIPTION
**Pull request by yomovh**
Not to merge before the actual update of the manager

### CODE REVIEW

File: pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.en-gb.md [hunk 1]
```diff
@@ -1,48 +1,79 @@
...
+- `New Public IP`: this option will create a new Floating IP for your Load Balancer. 
+- If you already have a Floating IP, the interface will offer you to pick one.
+
+If your Load Balancer is processing private traffic, choose `No Public IP`.
+
+#### Step 4 : Select the private network and the subnet where the Load Balancer will be spawned
+
+![Network choice](images/private_network.png){.thumbnail}
+
+The interface will inform you if the private network / subnet is not compliant with the prerequisites (see [Public Cloud Networking concepts](concepts-01-public-cloud-networking-concepts/guide.en-gb.md###Network pre requisites)).
```
Comment (OVH-Team-Guides) [inline pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.en-gb.md]: This link won't work as:
- format is not compliant (see the other links for reference)
- the target (Network pre requisites) does not exist

**Review by OVH-Team-Guides — CHANGES_REQUESTED** [review]

**Review by OVH-Team-Guides — APPROVED** [review]

File: pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md [hunk 2]
```diff
@@ -13,12 +13,70 @@ Notre Load Balancer Public Cloud est basé sur le service [Openstack Octavia](ht
...
+
+#### Étape 2 : Choix de la région
+
+![Choix de la région](images/region.png){.thumbnail}
+
+Seules les régions sur lesquelles vous avez un réseau privé et au moins un sous-réseau peuvent être sélectionnées. Sélectionnez la région et cliquez sur `Suivant`{.action}.
+
+#### Étape 3 : attacher une IP publique (ou non)
+
+![Public IP choice](images/IP_flottante.png){.thumbnail}
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md]: image name seems weird

File: pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md [hunk 3]
```diff
@@ -13,12 +13,70 @@ Notre Load Balancer Public Cloud est basé sur le service [Openstack Octavia](ht
...
+À ce stade, vous devez savoir si votre Load Balancer recevra du trafic public ou non (pour plus de détails, consultez notre page « [Concepts - Réseau Public Cloud](/pages/public_cloud/public_cloud_network_services/concepts-01-public-cloud-networking-concepts)) ». 
+
+Si votre Load Balancer reçoit du trafic public, vous disposez de 2 options :
+
+- `Nouvelle IP Publique` : cette option créera une nouvelle adresse Floating IP pour votre Load Balancer. 
+- Si vous avez déjà une adresse Floating IP, l'interface vous proposera d'en choisir une.
+
+Si votre Load Balancer ne doit recevoir que du trafic privé, choisissez « Aucune IP publique ».
+
+#### Étape 4 : sélectionnez le réseau privé et le sous-réseau sur lesquels le Load Balancer sera généré
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md]: sera créé

File: pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md [hunk 4]
```diff
@@ -13,12 +13,70 @@ Notre Load Balancer Public Cloud est basé sur le service [Openstack Octavia](ht
...
+
+![Choix du réseau](images/private_network.png){.thumbnail}
+
+L'interface vous informera si le réseau privé ou le sous-réseau ne sont pas conformes aux prérequis (voir notre page « [Concepts - Load Balancer](/pages/public_cloud/public_cloud_network_services/concepts-03-loadbalancer#network-prerequisites) »).
+
+#### Étape 5 (facultatif) : définir le ou les listeners et les membres
+
+![Choix du listener](images/listener.png){.thumbnail}
+
+- Dans un premier temps, choisissez le protocole et le portdu listener en fonction du trafic que vous allez recevoir. Notez qu'un listener spécifique appelé `Prometheus` est disponible pour monitorer votre Load Balancer. Dans ce cas, il n'est pas possible d'ajouter des membres. Pour plus d'informations sur ce processus d'écoute, consultez [cette page](/pages/public_cloud/public_cloud_network_services/technical-resources-02-octavia-monitoring-prometheus).
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md]: 1) portdu
2) processus d'écoute ?

File: pages/public_cloud/public_cloud_network_services/concepts-03-loadbalancer/guide.fr-fr.md [hunk 5]
```diff
@@ -43,6 +79,66 @@ Une politique L7 est l'association entre une ou plusieurs règles L7 et un liste
...
+
+| Type d'architecture | Réseau privé & sous-réseau | IP de passerelle définie dans les attributs de sous-réseau | Nombre d'adresses IP « libres » dans le sous-réseau | Composant Gateway & Floating IP \* |
+|---|---|---|---|---|
+|Private to Private | Obligatoire | Non requis | 3 ([src](/pages/public_cloud/public_cloud_network_services/known-limits)) | Non requis |
+|Public to Private | Obligatoire | Obligatoire | 5 ([src](/pages/public_cloud/public_cloud_network_services/known-limits)) | Obligatoire |
+|Public to Public | Obligatoire | Obligatoire | 5 ([src](/pages/public_cloud/public_cloud_network_services/known-limits)) | Obligatoire |
+
+\* : selon l'interface de contrôle que vous choisissez, soit ces composants seront créés pour vous (via l'espace client OVHcloud), soit vous devrez les créer spécifiquement (CLI OpenStack, Terraform ou GUI).
+
+## Statuts de fonctionnement et de provisioning <a name="operating-provisioning-status"></a>
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/concepts-03-loadbalancer/guide.fr-fr.md]: In the OVHCloud Control Plane, we decided to **not translate** provisioning / operating  status .

**Review by yomovh — COMMENTED** [review]
fr proof read

File: pages/public_cloud/public_cloud_network_services/concepts-03-loadbalancer/guide.fr-fr.md (hunk 5 above)
Comment (OVH-Team-Guides) [reply pages/public_cloud/public_cloud_network_services/concepts-03-loadbalancer/guide.fr-fr.md]: Thanks for the information. I'll edit the FR version accordingly.

File: pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md (hunk 2 above)
Comment (OVH-Team-Guides) [reply pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md]: Oops, this shouldn't have been translated. I'll fix this.

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @yomovh

**Pull request merged**